app/utils/jwt.ts

```diff
@@ -10,7 +10,7 @@ import { parseJWTPayload, parseJWTHeader } from './helpers';
 const h8 = 8 * 60 * 60;
 
 export const verifyToken = async (token: string) => {
-  if (!token) return false;
+  if (!token) return [undefined, 'e02'];
```

Comment:
@junminahn here no need to explicitly pass `undefined` as if you leave it blank, it would still consider it as `undefined`.

Reply:
@NithinKuruba right, I forgot about that; updated the pr.